service/frontend/workflowHandler.go

```diff
@@ -118,6 +118,7 @@ var (
 	errInvalidTaskStartToCloseTimeoutSeconds      = &gen.BadRequestError{Message: "A valid TaskStartToCloseTimeoutSeconds is not set on request."}
 	errClientVersionNotSet                        = &gen.BadRequestError{Message: "Client version is not set on request."}
 	errQueryDisallowedForDomain                   = &gen.BadRequestError{Message: "Domain is not allowed to query, please contact cadence team to re-enable queries."}
+	errUnauthorized                               = &gen.BadRequestError{Message: "Request unauthorized."}
```

Comment:
this doesn't belong here. Please move this within your decorator (authHandlerImpl) handler file